### PR TITLE
New version: twpayne.chezmoi version 2.34.2

### PR DESCRIPTION
**Pull request by twpayne**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/110014&drop=dogfoodAlpha

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
`x86` Automatic Validation ended with:
> Launching executable file: C:\Users\Validator\AppData\Local\Microsoft\WinGet\Packages\twpayne.chezmoi__DefaultSource\chezmoi.exe
> Exception during executable launch operation System.AggregateException: One or more errors occurred. (The specified executable is not a valid application for this OS platform.)

Is this a 64.bit executable?

File: manifests/t/twpayne/chezmoi/2.34.2/twpayne.chezmoi.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+PackageVersion: 2.34.2
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: chezmoi.exe
+  PortableCommandAlias: chezmoi
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/twpayne/chezmoi/releases/download/v2.34.2/chezmoi_2.34.2_windows_amd64.zip
+  InstallerSha256: 452B053A995154315B3A95569F5ADFC5343C2C6F97A036CDAD426C3E50FDDBE6
```
Comment (stephengillie) [inline manifests/t/twpayne/chezmoi/2.34.2/twpayne.chezmoi.installer.yaml:14]:
```suggestion
```

File: manifests/t/twpayne/chezmoi/2.34.2/twpayne.chezmoi.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- Architecture: x64
+  InstallerUrl: https://github.com/twpayne/chezmoi/releases/download/v2.34.2/chezmoi_2.34.2_windows_amd64.zip
+  InstallerSha256: 452B053A995154315B3A95569F5ADFC5343C2C6F97A036CDAD426C3E50FDDBE6
+- Architecture: arm
+  InstallerUrl: https://github.com/twpayne/chezmoi/releases/download/v2.34.2/chezmoi_2.34.2_windows_amd64.zip
+  InstallerSha256: 452B053A995154315B3A95569F5ADFC5343C2C6F97A036CDAD426C3E50FDDBE6
+- Architecture: arm64
+  InstallerUrl: https://github.com/twpayne/chezmoi/releases/download/v2.34.2/chezmoi_2.34.2_windows_amd64.zip
+  InstallerSha256: 452B053A995154315B3A95569F5ADFC5343C2C6F97A036CDAD426C3E50FDDBE6
```
Comment (stephengillie) [inline manifests/t/twpayne/chezmoi/2.34.2/twpayne.chezmoi.installer.yaml:23]:
```suggestion
```

File: manifests/t/twpayne/chezmoi/2.34.2/twpayne.chezmoi.installer.yaml (hunk 1 above)
Comment (twpayne) [reply manifests/t/twpayne/chezmoi/2.34.2/twpayne.chezmoi.installer.yaml:14]: Thank you very much for the review and the suggested changes.

Clearly there is a mismatch between the `Architecture` field (`x86` in this case) and the architecture in the zip file name (`amd64`). This error is also present in the other `InstallerUrl`s and `InstallSha256`s as you identify below.

This PR was automatically generated by https://github.com/vedantmgoyal2009/winget-releaser, so I will raise an issue with the automation.

Comment (twpayne) [reply manifests/t/twpayne/chezmoi/2.34.2/twpayne.chezmoi.installer.yaml:14]: Opened https://github.com/vedantmgoyal2009/winget-releaser/issues/173.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**